configure.ac

```diff
@@ -49,6 +49,7 @@ AC_PROG_INSTALL
 AC_PROG_CC
 AC_PROG_LIBTOOL
 AM_PROG_AS
+AM_PROG_CC_C_O
```

Comment:
"This is an obsolescent macro that checks that the C compiler supports the -c and -o options together. Note that, since Automake 1.14, the AC_PROG_CC is rewritten to implement such checks itself, and thus the explicit use of AM_PROG_CC_C_O should no longer be required."
-- https://www.gnu.org/software/automake/manual/html_node/Public-Macros.html

Was the lack of this ./configure test a real problem for you? If so, which distro and compiler?